src/wms/src/test/java/org/geoserver/wms/map/MarkFactoryProcessorProviderFactoryTest.java

```diff
@@ -0,0 +1,98 @@
+package org.geoserver.wms.map;
```

Comment:
header required on new file, surprised PMD or check style did not catch this?

Reply:
Right, header added.